doc/reference/reference_lua/box_info/package.rst

```diff
@@ -0,0 +1,15 @@
+.. _box_info_package:
+
+================================================================================
+box.info.package
+================================================================================
+
+.. module:: box.info
+
+.. data:: package
+
+    The package name.
```

Comment:
Nit: We can say something like the following:

> Currently there are `Tarantool` and `Tarantool Enterprise` packages.

This way it is less abstract and a developer knows that it allows to distinguish CE/EE editions.